pipeline/profile-controller/base/composite-controller.yaml

```diff
@@ -17,6 +17,18 @@ spec:
     resource: configmaps
     updateStrategy:
       method: OnDelete
+  - apiVersion: v1
+    resource: serviceaccounts
+    updateStrategy:
+      method: OnDelete
```

Comment:
I used `updateStrategy: OnDelete` because users may want to configure those secrets/configmaps.

I think for visualization server, we should always control it. So we can use the default updateStrategy instead.

Reply:
Done